README.md

```diff
@@ -1 +1,85 @@
 # draft-store-client
+
+[![Greenkeeper badge](https://badges.greenkeeper.io/hmcts/cmc-citizen-frontend.svg)](https://greenkeeper.io/)
+
+[![Standard - JavaScript Style Guide](https://img.shields.io/badge/code%20style-standard-brightgreen.svg)](http://standardjs.com/)
+
+This is the draft store client module to access draft store. 
+Below are steps to configure draft store clients
+
+* User need to implement ServiceAuthTokenFactory interface to access all client modules.  
```

Comment:
Implement the `ServiceAuthTokenFactory` interface

Reply:
updated